### ISSUE TITLE
encfs idle parameter is confusing; possibly working incorrectly

### ISSUE THREAD

**Issue by j1mw3b** [vgough/encfs#612]
Is this the intended way for the encfs timeout to work?
I set a idle time of - say 300 minutes (5 hours).  I then stop working on the encrpted folder files and my laptop goes to sleep.  Next day, encfs still has the folder unencrypted.
Seems it only checks the number of seconds/minutes have elapsed and does not care what time of day it is.
I would call this a security issue since obviously 5 hours have passed, but since the machine was suspended, encfs doesn't know it.  
I think the idle thread needs to check the time-of-day also in addition to elapsed seconds of "awake" time.
Unfortunately, I only a c guy, not c++ (purely hate it) and can't try to fix it.  If I am wrong, please just delete this request.

**Comment by benrubson** [comment]
You're right, we could use current timestamp instead of the elapsed number of seconds alive.
